src/tests/unit/auto/auto_startup_fallback_properties_test.cpp

```diff
@@ -0,0 +1,157 @@
+        metaDevices.clear();
+        inferReqInternal.reset();
+    }
+
+    void SetUp() override {
+       // prepare mockExeNetwork
+       auto mockIExeNet = std::make_shared<NiceMock<MockIExecutableNetworkInternal>>();
+       mockExeNetwork = {mockIExeNet, {}};
+       // prepare mockicore and cnnNetwork for loading
+       core = std::make_shared<NiceMock<MockICore>>();
+       NiceMock<MockMultiDeviceInferencePlugin>* mock_multi = new NiceMock<MockMultiDeviceInferencePlugin>();
+       plugin.reset(mock_multi);
+       function = ngraph::builder::subgraph::makeConvPoolRelu();
+       cnnNet = InferenceEngine::CNNNetwork(function);
+       // replace core with mock Icore
+       plugin->SetCore(core);
+       // mock execNetwork can work
+       inferReqInternal = std::make_shared<NiceMock<MockIInferRequestInternal>>();
+       ON_CALL(*mockIExeNet.get(), CreateInferRequest()).WillByDefault(Return(inferReqInternal));
+       IE_SET_METRIC(OPTIMAL_NUMBER_OF_INFER_REQUESTS, optimalNum, 1);
+       ON_CALL(*mockIExeNet.get(), GetMetric(StrEq(METRIC_KEY(OPTIMAL_NUMBER_OF_INFER_REQUESTS))))
+           .WillByDefault(Return(optimalNum));
+       IE_SET_METRIC(SUPPORTED_CONFIG_KEYS, supportConfigs, {});
+       ON_CALL(*core, GetMetric(_, StrEq(METRIC_KEY(SUPPORTED_CONFIG_KEYS)), _))
+           .WillByDefault(Return(supportConfigs));
+       ON_CALL(*core, GetConfig(_, StrEq(GPU_CONFIG_KEY(MAX_NUM_THREADS))))
+           .WillByDefault(Return(12));
+    }
+};
+
+TEST_P(AutoStartupFallback, releaseResource) {
+    // get Parameter
+    bool startup_fallback;
+    Config config;
+    std::tie(startup_fallback, config) = this->GetParam();
+    // test auto plugin
+    plugin->SetName("AUTO");
+
+    ON_CALL(*core, LoadNetwork(::testing::Matcher<const InferenceEngine::CNNNetwork&>(_),
+                ::testing::Matcher<const std::string&>(_),
+                ::testing::Matcher<const Config&>(_))).WillByDefault(Return(mockExeNetwork));
+
+    metaDevices = {{CommonTestUtils::DEVICE_CPU, {}, -1}, {CommonTestUtils::DEVICE_GPU, {}, -1}};
+    // DeviceInformation devInfo;
+    ON_CALL(*plugin, ParseMetaDevices(_, _)).WillByDefault(Return(metaDevices));
+    ON_CALL(*plugin, GetValidDevice)
+        .WillByDefault([this](const std::vector<DeviceInformation>& metaDevices, const std::string& netPrecision) {
+            std::list<DeviceInformation> devices(metaDevices.begin(), metaDevices.end());
+            return devices;
+        });
+    ON_CALL(*plugin, SelectDevice(_, _, _)).WillByDefault(Return(metaDevices[1]));
+
+    EXPECT_CALL(
+        *core,
+        LoadNetwork(::testing::Matcher<const InferenceEngine::CNNNetwork&>(_),
+                    ::testing::Matcher<const std::string&>(CommonTestUtils::DEVICE_GPU),
+                    ::testing::Matcher<const std::map<std::string, std::string>&>(_)))
+        .Times(1);
+    if (startup_fallback) {
+        std::map<std::string, std::string> test_map = {{"PERFORMANCE_HINT", "LATENCY"}};
```

Comment:
Can you explain why set "LATENCY" mode here.

Reply:
When the CPU as an accelerator, PERFORMANCE_HINT must be LATENCY.